docs/deploy-rbd.md

```diff
@@ -156,3 +157,51 @@ The Helm chart is located in `charts/ceph-csi-rbd`.
 **Deploy Helm Chart:**
 
 [See the Helm chart readme for installation instructions.](../charts/ceph-csi-rbd/README.md)
+
+## Encryption for RBD volumes
```

Comment:
@vasyl-purchel we should list the `prereq` to be met in the node for supporting encryption , example : binaries to be present, the kernel module to be loaded..etc

Reply:
so regarding binaries - there is no requirements, encryption is done inside rbd-plugin container that have all binaries included, the only requirement probably needed is dm-crypt module in kernel (that was missing previously from minikube image), will add the note on kernel module

Reply:
Sure, `dm-crypt` is a must for encryption to work here, so we should definitely mention it. Also there are different flavors of OS which is kind of stripped for container host platform. I am not sure everyone carry this module in it though.

Reply:
note about kernel module is added